### DISCUSSION TITLE
Fatal error

### DISCUSSION THREAD

**Discussion by pazao** [SafeExamBrowser/seb-win-refactoring#370] [Q&A]
Hi
My student installed seb, but when he started the exam, there was a fatal error.
He couldn't find the log file either.
Could you please inform me of what might have gone wrong and how to fix it?
![Fatal Error](https://user-images.githubusercontent.com/87844542/156989551-7318ab0d-ffae-46de-aa29-6253fa1a2124.png)
![Untitled](https://user-images.githubusercontent.com/87844542/156989594-ee061e75-219b-4770-a5d8-59b59c20cb45.png)

**Comment by dbuechel** [comment]
Which SEB version are you using? The newest version should have fixed this issue, please update SEB if you're not yet using the latest release version.

The log files definitely are located at `%LocalAppData%\SafeExamBrowser\Logs`, please make sure that you're using the correct user name (i.e. the one under which you executed SEB).

**Reply by pazao** [reply]
Thank you for your answer. I've noted the information.
